app/views/jokes.py

```diff
@@ -34,3 +35,39 @@ def joke(joke_id):
     current_joke = Joke.query.filter_by(id=joke_id).first()
     return render_template('jokes/joke.html',
                            joke=current_joke)
+
+@JOKES_MOD.route('/rate-joke', methods=['POST'])
+def rate_joke():
+    """add new reaction to database"""
+    joke_id = request.form["joke_id"]
+    reaction_id = request.form["reaction_id"]
+    try:
+        reaction_id = int(reaction_id)
+    except:
+        return False
```

Comment:
Is this case possible under normal conditions? If no, at least log it so you can debug what's happening